OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -939,37 +762,30 @@ void BuildUnit(string category, string name)
 
 		void INotifyDamage.Damaged(Actor self, AttackInfo e)
 		{
-			if (!IsEnabled || e.Attacker == null)
-				return;
-
-			if (e.Attacker.Owner.Stances[self.Owner] == Stance.Neutral)
+			if (!IsEnabled)
 				return;
 
-			var rb = self.TraitOrDefault<RepairableBuilding>();
+			// TODO: Add an option to include this in CheckSyncUnchanged.
+			// Checking sync for this is too expensive to include it by default,
+			// so it should be implemented as separate sub-option checkbox.
+			using (new PerfSample("attackResponse_bots"))
```

Comment:
Style nit: I don't think we use camelcase anywhere else in perf sample names.  It might be better to rename this to `bot_attack_response` and `tick_bots` to `bot_tick`?

Reply:
Done.